kernel/kernel-default/src/main/java/io/delta/kernel/parquet/ParquetConverters.java

```diff
@@ -530,6 +568,142 @@ public void resizeIfNeeded()
         }
     }
 
+    public abstract static class BaseDecimalConverter extends BasePrimitiveColumnConverter {
```

Comment:
Looks like these converters can be in their own file just like the Map/Array/Struct converters.